src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceProtectorTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
+using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
+{
+    public class ManagedInstanceProtectorTests : SqlTestsBase
+    {
+        public ManagedInstanceProtectorTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact(Skip = "Skip due to long setup time for managed instance")]
```

Comment:
Again, looks like all tests are skipped now, and we need one of these test running.  Also please remove the [Fact(Skip)] from all the tests that won't run on CI and just leave //[Fact]